torch/csrc/utils/python_dispatch.cpp

```diff
@@ -497,13 +499,18 @@ void initDispatchBindings(PyObject* module) {
          const char* file,
          uint32_t linenum) {
         HANDLE_TH_ERRORS
+        // Malloc and leak the file string to ensure it remains valid
+        // for the lifetime of the Library object
+        char* leaked_file = static_cast<char*>(malloc(strlen(file) + 1));
```

Comment:
Don't mix C and C++...
```suggestion
        auto leaked_file = new char[strlen(file) + 1];
```